htdocs/compta/bank/ligne.php

```diff
@@ -60,16 +60,16 @@
  * Actions
  */
 
-if ($user->rights->banque->consolidate && $action == 'dvnext')
+if ($user->rights->banque->consolidate)
```

Comment:
If we call the page with no action or any action, we will execute this code when we should not, wouldn't we ?
We should add test if $action start with dv for example.